src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebefore/InputNoWhitespaceBeforeLabelAndSwitch.java

```diff
@@ -0,0 +1,21 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespacebefore;
+
+class InputNoWhitespaceBeforeLabelAndSwitch
```

Comment:
Please add other cases where COLON can be used. For example, Ternary operator (`boolean ? true : false`), maybe something else.

Reply:
Other input files like `InputNoWhitespaceBeforeDefault` already cover this.
If you look at the `testDefault` it runs the check for all tokens, including `COLON`.
Rest assured no false positive is being reported.

Reply:
Done, I've added it anyway :)